packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -33,6 +33,7 @@ import { getRootWorkspacePath } from '../../../src/util';
 // tslint:disable:no-unused-expression
 describe('Force Project Create', () => {
   const PROJECT_NAME = 'sfdx-simple';
+  const PROJECT_NAME_WITH_LEADING_TRAILING_SPACES = '  sfdx-simple  ';
```

Comment:
nit: you could use a template literal here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

 =  \` ${PROJECT_NAME}  \` 

here